src/applications/static-pages/representative-status/components/App/index.jsx

```diff
@@ -29,56 +27,39 @@ export const App = ({
     useToggleLoadingValue,
     TOGGLE_NAMES,
   } = useFeatureToggle();
+
   const togglesLoading = useToggleLoadingValue();
 
   const appEnabled = useToggleValue(TOGGLE_NAMES.representativeStatusEnabled);
 
-  useEffect(
-    () => {
-      if (loggedIn) {
-        focusElement('.representative-status-widget');
-      }
-    },
-    [loggedIn],
-  );
-
-  if (togglesLoading || !appEnabled) {
-    return null;
-  }
+  // if (togglesLoading || !appEnabled) {
+  //   return null;
+  // }
 
   return (
     <>
-      {showIntroCopy && (
+      <Auth
+        DynamicHeader={DynamicHeader}
+        DynamicSubheader={DynamicSubheader}
+        useRepresentativeStatus={useRepresentativeStatus}
+      />
+
+      {/* {loggedIn ? (
```

Comment:
Is the commented out code for testing on staging?

Reply:
Yeesh, nice catch. Reverted